### ISSUE TITLE
Incorrect warning IDE0060 (Remove unused parameter) for interface implementation by base method

### ISSUE THREAD

**Issue by tmat** [dotnet/roslyn#48865]
**Version Used**: 
Version 16.9.0 Preview 1.0 [30614.203.main]

**Steps to Reproduce**:

```C#
class C { public void F(int x) { } }
class D : C, I { }
interface I { void F(int x); }
```

**Expected Behavior**:
No warning.

**Actual Behavior**:
Warning IDE0060: Remove unused parameter 'x'

**Comment by mavasani** [comment]
This is as per the design decision when implementing this analyzer. Avoiding these false positives would need compilation/solution analysis and skipping these cases would lead to extremely high number of false negatives. It was decided that users should explicitly mark such intentionally unused parameters as discards.
